src/main/java/com/google/sps/servlets/StorePromotionQuestion.java

```diff
@@ -0,0 +1,116 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.QuestionChoice;
+import com.google.sps.data.QuestionDatabase;
+import com.google.sps.data.QuizQuestion;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.PatternSyntaxException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Stores career question data in database */
+@WebServlet("/storePromotion")
+public class StorePromotionQuestion extends HttpServlet {
+  private static final String JSON_CONTENT_TYPE = "application/json";
+  private static final String FORM_SUBMIT_PARAMETER = "promotionQuizSubmit";
+  private static final String QUESTION_PARAMETER = "promotionQuestion";
+  private static final String CHOICETEXT_PARAMETER = "promotionChoiceText";
+  private static final String CHOICE_IS_ACCEPTED_PARAMETER = "promotionAcceptedChoice";
+  private static final String QUIZ_CAREERPATH_PARAMETER = "promotionCareerPath";
+  private static final String QUIZ_LEVEL_PARAMETER = "promotionQuizLevel";
+  private static final String ACCEPTED_CHOICE = "y";
+  private static final String NOT_ACCEPTED_CHOICE = "n";
```

Comment:
There might be a better String than y or n

Reply:
I might keep it as "y" or "n" since it is a quick way to enter things into a form but I'm open for other suggestions!